index.bs

```diff
@@ -203,18 +203,26 @@ for tracking sites is cleared after 24 hours, unless the user has interacted
 with the site in the first-party context in the last 45 days.
 
 <h4 id="mitigations-brave">Brave</h4>
-Brave uses three list-based approaches to combat [=navigational tracking=].
+Brave uses four list-based approaches to combat [=navigational tracking=].
 
 First, Brave strips query parameters commonly used for [=navigational tracking=]
 from URLs on navigation. This list is maintained by Brave.
 
-Second, in the non-default, "aggressive blocking" configuration, Brave uses
+Second, by default, when i) the user is about to visit a list-identified
+bounce-tracking site, and ii) the current profile does not contain any cookies
+or DOM state for that site, Brave will create a new, "ephemeral", empty storage
```

Comment:
By "DOM state", do you mean a specific set of endpoints from https://storage.spec.whatwg.org/#storage-endpoints, or just anything stored in any of the site's [storage bottles](https://storage.spec.whatwg.org/#storage-bottle)?

Reply:
The former (thanks for the clarification!)

Reply:
Perhaps
```suggestion
or {{WindowLocalStorage/localStorage}} for that site, Brave will create a new, "ephemeral", empty storage
```

or
```suggestion
or storage in the "localStorage" or "sessionStorage" [=storage endpoints=] for that site, Brave will create a new, "ephemeral", empty storage
```
(or whatever set of storage endpoints you're actually checking.